Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "dummy"
+version = "0.0.0"
+
+[lib]
+path = "empty.rs"
```

Comment:
Are all of the above lines necessary ?
Go ahead and use “deno” instead of “dummy” ... maybe add a comment here about how this is only used for dependency management and called from tools/build_third_party.py only when adding deps

Reply:
The [lib] stuff is not. Forgot to remove it when I was experimenting.
Will add the comment.